postgis/lwgeom_transform.c

```diff
@@ -225,3 +226,215 @@ Datum LWGEOM_asKML(PG_FUNCTION_ARGS)
 		PG_RETURN_TEXT_P(kml);
 	PG_RETURN_NULL();
 }
+
+
+/********************************************************************************
+ * PROJ database reading functions
+ */
+
+
+#if POSTGIS_PROJ_VERSION >= 61
+
+struct srs_entry {
+	text* auth_name;
+	text* auth_code;
+};
+
+struct srs_data {
+	struct srs_entry* entries;
+	uint32_t num_entries;
+	uint32_t capacity;
+	uint32_t current_entry;
+};
+
+static Datum
+srs_tuple_from_entry(const struct srs_entry* entry, TupleDesc tuple_desc)
+{
+	HeapTuple tuple;
+	Datum tuple_data[4] = {0, 0, 0, 0};
+	bool  tuple_null[4] = {false, false, false, false};
+	PJ_CONTEXT *ctx = NULL;
+	const char* const empty_options[2] = {NULL};
+	const char* const wkt_options[2] = {"MULTILINE=NO", NULL};
```

Comment:
you don't need to specify the size:
```suggestion
	const char* const empty_options[] = {NULL};
	const char* const wkt_options[] = {"MULTILINE=NO", NULL};
```